spec/lib/staging_mail_interceptor_spec.rb

```diff
@@ -0,0 +1,77 @@
+require "rails_helper"
+
+RSpec.describe StagingMailInterceptor do
+  let(:to) { [] }
+  let(:interceptor) { StagingMailInterceptor.new(message) }
+  subject(:message) { OpenStruct.new to: to, subject: "A message" }
+
+  describe "whitelisting" do
+    let(:whitelist) { ["allowed@example.org", "allowed2@example.org", "allowed3@example.org"] }
+    let(:send_to) { ["sendto@example.org"] }
+
+    before do
+      allow(interceptor).to receive(:whitelist) { whitelist }
+      allow(interceptor).to receive(:send_to_addresses) { send_to }
+      interceptor.process
+    end
+
+    describe "when the email is sent to someone on the list" do
+      describe "single address" do
+        let(:to) { ["allowed@example.org"] }
```

Comment:
If I change this to `allowed@EXAMPLE.ORG` the spec fails. Should it be case-insensitive?